src/data/custom_icons.ts

```diff
@@ -1,7 +1,8 @@
 import { customIconsets } from "./custom_iconsets";
 
 export interface CustomIcon {
-  path: string;
+  path?: string;
```

Comment:
`path` should not be optional I think?

Reply:
resolved in commit "991bfb59c5bf6a92c4fe9ea8a893c9c89003a7a6".